apps/emqx_management/i18n/emqx_mgmt_api_publish_i18n.conf

```diff
@@ -124,4 +124,55 @@ MQTT 消息发布的错误码，这些错误码也是 MQTT 规范中 PUBACK 消
             zh: "失败的详细原因。"
         }
     }
+    message_properties {
+        desc {
+             en: "The Properties of the PUBLISH message."
+             zh: "PUBLISH 消息里的 Property 字段。"
+        }
+    }
+    msg_payload_format_indicator {
+        desc {
+             en: """0 (0x00) Byte Indicates that the Payload is unspecified bytes, which is equivalent to not sending a Payload Format Indicator.
+
+1 (0x01) Byte Indicates that the Payload is UTF-8 Encoded Character Data. The UTF-8 data in the Payload MUST be well-formed UTF-8 as defined by the Unicode specification and restated in RFC 3629.
```

Comment:
I feel that this sentence is a bit redundant, maybe there is no need to explain what utf8 is

Reply:
See below, it's copied from old docs